### PR TITLE
Add HTML setup

### PR DESCRIPTION
**Pull request by RandyGitProjects**
This is our basic HTML setup.

### CODE REVIEW

**Review by brendel-r — COMMENTED** [review]
Looks great!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                    <h3>Idea title</h3>
+                    <p>Idea body</p>
+                </div>
+                <div class="bottom-bar">
+                    <button class="comment-btn"></button>
+                    <label for="comment-btn">Comment</label>
+                </div>
+            </section>
+        </section>
+
```
Comment (asutherland91) [inline index.html:42]: Line here should be deleted.

**Review by Denzel-Turambi — COMMENTED** [review]
Looks good!

**Review by asutherland91 — COMMENTED** [review]
Everything looks good thus far! You might want to consider making your tab distance a little less

**Pull request merged**